From 30cee9cd4902a72fcef06f85867ba136a6d9d16c Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Sun, 06 Mar 2022 17:32:38 +0200
Subject: [PATCH] [SW-69581] gaudi3: adjust i2c mask reg names to latest specs

I2C mask register was relocated in specs headers, and the name
was changed. Code is adjusted respectively.

Change-Id: I3b8721a2aa96492adb5123e2bc6b03507f41d453
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 6aed290..0396a40 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -17,6 +17,7 @@
 #include <d0_psoc_uart_0_regs.h>
 #include <d0_parc_arc0_cfg_regs.h>
 #include <d0_psoc_i2c_s_regs.h>
+#include <d0_parc_int_aggr_i2c_slv_comb_regs.h>
 #include <gaudi3/gaudi3_boot_info.h>
 
 #include <bitops.h>
@@ -243,7 +244,7 @@
 	global_conf.prstn_reg = mmD0_PSOC_GLOBAL_CONF_PRSTN;
 	global_conf.kmd_msg_to_cpu_reg = mmD0_PSOC_GLOBAL_CONF_KMD_MSG_TO_CPU;
 	global_conf.i2c_slv_reg = mmD0_PSOC_GLOBAL_CONF_I2C_SLV;
-	global_conf.i2c_slv_intr_mask_reg = mmD0_PSOC_GLOBAL_CONF_I2C_SLV_INTR_MASK;
+	global_conf.i2c_slv_intr_mask_reg = mmD0_PARC_INT_AGGR_I2C_SLV_COMB_MASK;
 	global_conf.cpu_boot_status_reg = mmD0_PSOC_GLOBAL_CONF_CPU_BOOT_STATUS;
 	global_conf.cpu_cmd_status_to_host_map = mmCPU_CMD_STATUS_TO_HOST;
 	global_conf.cpu_boot_err0_map = mmCPU_BOOT_ERR0;
